src/main/scala/shine/DPIA/Compilation/ContinuationTranslation.scala

```diff
@@ -62,8 +59,7 @@ object ContinuationTranslation {
           `if`(x) `then` con(thenP)(C) `else` con(elseP)(C)
         })
 
-      case Proj1(_) => throw new Exception("This should never happen")
-      case Proj2(_) => throw new Exception("This should never happen")
+      case Proj1(_) | Proj2(_) => C(E)
```

Comment:
Where is this change coming from?

Reply:
As observed by you at the top, the evaluation order has changed now and this here is to allow the variables, i.e. (`accum.rd` and `accum.wr`) to work.

Reply:
I thought we had a reason not to have a continuation translation for these .. but I am not sure I remember why. Maybe it is because such projections should only be introduced by the translation, and not exist before it? I guess now the translation ends up introducing projections and throwing them back at its own recursion, which did not happen before.